### ISSUE TITLE
Investigations could contain multiple Studies?

### ISSUE THREAD

**Issue by mccalluc** [refinery-platform/refinery-platform#1928]
My sense is that in the ISA model, an Investigation could contain multiple studies. I think our database schema would work that way... but I'm not actually sure that we support it that well:

Currently we have:
```python
    def get_latest_study(self, version=None):
        try:
            return Study.objects.get(
                investigation=(
                    self.get_latest_investigation_link(version).investigation
                )
            )
        except(Study.DoesNotExist, Study.MultipleObjectsReturned) as e:
            raise RuntimeError("Couldn't properly fetch Study: {}".format(e))
```
```python
        analysis_config = {
            "custom_name": "Analysis: {}".format(self),
            "studyUuid": self.dataset.get_latest_study().uuid,
            "toolUuid": self.uuid,
            "user_id": self.get_owner().id,
            "workflowUuid": self.tool_definition.workflow.uuid
        }
```

(and many tests that assume there would be only one.)

I feel like this could be a can of worms, but wanted to raise the question.

**Comment by ngehlenborg** [comment]
Yes, per ISA specs an Investigation can contain multiple Studies and we need to support that.

We don't have any actual examples of such ISA-Tabs in the Stem Cell Commons, but they exist.

**Comment by mccalluc** [comment]
This had been noted earlier: #1001. Seems like they're not quite duplicates, though that could be argued.

**Comment by mccalluc** [comment]
Seems we have the same problem between Studies and Assays: retitle to reflect that.

**Comment by SarahKit** [comment]
Test file here: https://beta.genometranslationcommons.org/media/file_store/2b/62/BII-I-1_archive.zip

**Comment by drj11** [comment]
[BII-I-1_archive.zip](https://github.com/refinery-platform/refinery-platform/files/1244620/BII-I-1_archive.zip)

**Comment by drj11** [comment]
I think this is the one of the files in the ISA-Creator distribution, right @SarahKit? Therefore we can redistribute it reasonably freely.

**Comment by jkmarx** [comment]
I closed #1001. After we test the back end support, we need to update multiple features old launch pad (#1001), new dashboard, and file browser. Priority wise the new dashboard and file browser would take precedence. (I believe.) I believe updating the preview and explore would be very involved.
